src/Raven.Server/Utils/CertificateUtils.cs

```diff
@@ -69,7 +68,6 @@ internal static bool CertHasKnownIssuer(X509Certificate2 userCertificate, X509Ce
                 issuerCertificate = userChain.ChainElements.Count > 1
                     ? userChain.ChainElements[1].Certificate
                     : userChain.ChainElements[0].Certificate;
-                issuerPinningHash = issuerCertificate.GetPublicKeyPinningHash();
```

Comment:
I doubt that, but wanted to confirm. That public pinning hash trust mechanism related to trusting replication after cert renewal on the source is not touched by this PR, right?

Reply:
It is not touched, this is about having a well known pinning hash, which we aren't using and probably don't want to have in the first place.